app/docs/0.11.x/clustering.md

```diff
@@ -4,135 +4,54 @@ title: Clustering Reference
 
 # Clustering Reference
 
-Multiple Kong nodes pointing to the same datastore **must** belong to the same "Kong Cluster". 
+Multiple Kong nodes using the same datastore belong to the same "Kong Cluster". 
```

Comment:
i see this is pre existing, but, lets take this opportunity to lowercase 'cluster'. nowhere else in this doc is cluster capitalized.